.github/workflows/axosyslog-docker.yml

```diff
@@ -151,11 +153,30 @@ jobs:
 
       - name: Extract metadata (AxoSyslog version) for Docker
         id: docker-metadata-tags
-        uses: docker/metadata-action@v5
-        with:
-          images: ${{ needs.prepare.outputs.image-name }}
-          tags: type=match,pattern=axosyslog-([0-9].*),group=1
-          sep-tags: ','
+        run: |
+          previous_docker_tag=$(skopeo inspect docker://ghcr.io/axoflow/axosyslog:latest | jq '.RepoTags' | sed 's/ \|"\|,//g' | grep "^[0-9]*.[0-9]*.[0-9]*$\|^[0-9]*.[0-9]*.[0-9]*-[0-9]*$" | tail -n 1)
+          previous_docker_tag_short=$(echo $previous_docker_tag | cut -d"-" -f1)
+          previous_docker_tag_patch=$(echo $previous_docker_tag | cut -d"-" -f2)
+          current_git_tag=$(echo ${{ github.ref }} | cut -d"-" -f2)
```

Comment:
We need to make sure everything below this line is skipped when github.ref is not a version tag.
We use this job to produce nightly and snapshot builds as well.

Reply:
Thanks, fixed.